scheduler/kubernetes/upgrade.md

```diff
@@ -8,57 +8,115 @@ sidebar: home_sidebar
-    metadata:
-       name: node-get-put-list-role
-    rules:
-    - apiGroups: [""]
-      resources: ["nodes"]
-      verbs: ["get", "update", "list"]
-    - apiGroups: [""]
-      resources: ["pods"]
-      verbs: ["get", "list"]
-    EOF
-    ```
-
-### 3. Upgrade Portworx pods
-
-It is not recommended to delete the Portworx pod while an application is actively issuing I/O. This can induce race conditions in docker causing it to hang. 
-
-The following procedure should be followed:
-1. Cordon the node where you want to upgrade Portworx: `$ kubectl cordon <node-name>`
-2. Delete application pods running on this node that are using Portworx volumes: `$ kubectl delete pod <pod-name>`
-    * Since application pods are expected to be managed by a controller like `Deployement` or `StatefulSet`, Kubernetes will spin up a new replacement pod on another node.
-3. Delete Portworx pod running on this node. This will start a new Portworx pod on this node with the new version you set above. (Note: Cordoning a kubernetes node doesn't affect DaemonSet pods)
-4. A new Portworx pod with the new version will be initiated on this node. This pod will stay in initializing state.
-5. Reboot the node. (This step is needed only if your current Portworx version is 1.2.9 since it requires a reboot of the host to perform upgrade of our kernel driver.)
-6. Uncordon the node once it comes up.
+This guide describes the procedure how to upgrade Portworx in Kubernetes environment as OCI container, which is the default and recommended method of running Portworx in Kubernetes.
+
+We assume you have deployed Portworx as [Kubernetes Daemonset](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) using instructions provided at our [install page](/scheduler/kubernetes/install.html).
+
+
+>**IMPORTANT:**<br/>We do not recommend upgrading Portworx using [Kubernetes instructions](https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/) (e.g. via `kubectl set image ds/portworx portworx=portworx/XXXX:### -n kube-system`).<br/>
+>Instead, whenever possible, we recommend using the [install.portworx.com](http://install.portworx.com) site to generate latest recommended YAML-spec, and re-apply the spec to upgrade Portworx.
+
+
+>**NOTE**: This procedure will also automatically migrate all legacy PX-Container deployments into the OCI containers.  If you are looking for legacy instructions how to upgrade PX-Container deployments, you can find them [here](/scheduler/kubernetes/upgrade-legacy.html).
+
+
+## Upgrading PX-OCI
+
+The PX-OCI Daemonset is using `RollingUpdate` update strategy, which greatly simplifies the upgrade process.
+
+### Step 1) Apply updated YAML-spec
+
+The upgrade the PX-OCI, we will just have to re-apply the YAML spec-file generated from the [install.portworx.com](http://install.portworx.com) site.  The same applies if we want to migrate from the deprecated PX-Container to the recommended PX-OCI daemonset.
```

Comment:
Remove the line "The same applies if we want to migrate from the deprecated PX-Container to the recommended PX-OCI daemonset.".

Good to have as less text as possible.